src/main/scala/no/ndla/articleapi/service/converters/HTMLCleaner.scala

```diff
@@ -17,9 +17,13 @@ object HTMLCleaner extends ConverterModule with LazyLogging {
     removeComments(element)
     removeNbsp(element)
     removeEmptyTags(element)
-    val ingress = extractIngress(element)
 
-    (content.copy(content=jsoupDocumentToString(element), ingress=ingress),
+    val ingressLanguage = content.ingress match {
```

Comment:
Tror denne kan gjøres litt enklere.
Hvis `extractIngressText` endres til å ta i mot en `String`, ikke `Option[String]`, og returnere en `String` ikke en `Option[String]`, kan man for eksempel gjøre noe ala `content.ingress.map(ingress => LanguageIngress(extractIngressText(...))).getOrElese(extractIngress(element)`